src/app/fetcher/fullDomFetcher.test.js

```diff
@@ -0,0 +1,86 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import http from 'http';
+
+import fetch from './fullDomFetcher.js';
+import { InaccessibleContentError } from '../errors.js';
+
+const { expect } = chai;
+chai.use(chaiAsPromised);
+
+const SERVER_PORT = 8976;
+
+describe('FullDomFetcher', function () {
```

Comment:
Avoid linting errors.
```suggestion
describe('FullDomFetcher', () => {
```